arkouda/groupbyclass.py

```diff
@@ -263,9 +265,38 @@ def __init__(
                 self.keys, return_groups=True, assume_sorted=self.assume_sorted
             )
 
-        self.size = self.permutation.size
+        self.length = self.permutation.size
         self.ngroups = self.segments.size
 
+    def size(self) -> Tuple[groupable, pdarray]:
+        """
+        This alias for "count" was added to conform with Pandas API
```

Comment:
I would move this into a `Notes` section in the docstring. The descriptions here should be similar to count.